browser/html/cool.html.m4

```diff
@@ -311,6 +311,7 @@ m4_ifelse(MOBILEAPP,[true],
       window.enableWelcomeMessage = %ENABLE_WELCOME_MSG%;
       window.enableWelcomeMessageButton = %ENABLE_WELCOME_MSG_BTN%;
       window.userInterfaceMode = '%USER_INTERFACE_MODE%';
+      window.useIntegrationTheme = '%USE_INTEGRATION_THEME%';
```

Comment:
Here you should probably set window.useIntegrationTheme also in the true branch of the `m4_ifelse(MOBILEAPP,[true],` , a dozen lines earlier.